ner_v2/detectors/numeral/number/standard_number_detector.py

```diff
@@ -87,7 +90,14 @@ def detect_number(self, text):
         for detector in self.detector_preferences:
             number_list, original_list = detector(number_list, original_list)
             self._update_processed_text(original_list)
-        return number_list, original_list
+        sorted_number_list = [_pop_key_from_dict(num_dict, NUMBER_DETECTION_RETURN_DICT_SPAN) for num_dict in
```

Comment:
Please explain here too, what are we trying to sort and what are we popping here.

Reply:
We are sorting according to span in ascending order, and then popping Span key from dict as we don't want to send it further